rcgen/src/sign_algo.rs

```diff
@@ -145,19 +145,17 @@ pub(crate) mod algo {
 		params: SignatureAlgorithmParams::Null,
 	};
 
-	// TODO: not really sure whether the certs we generate actually work.
-	// Both openssl and webpki reject them. It *might* be possible that openssl
-	// accepts the certificate if the key is a proper RSA-PSS key, but ring doesn't
-	// support those: https://github.com/briansmith/ring/issues/1353
+	// openssl accepts the certificate if the key is a proper RSA-PSS key, but
+	// ring doesn't support those: https://github.com/briansmith/ring/issues/1353
 	//
 	/// RSA signing with PKCS#1 2.1 RSASSA-PSS padding and SHA-256 hashing as per [RFC 4055](https://tools.ietf.org/html/rfc4055)
-	pub(crate) static PKCS_RSA_PSS_SHA256: SignatureAlgorithm = SignatureAlgorithm {
+	pub static PKCS_RSA_PSS_SHA256: SignatureAlgorithm = SignatureAlgorithm {
```

Comment:
Should we add the `_SHA384` and `_SHA512` variants as well for parity with `PKCS_RSA_XXX`?

Reply:
Done